tests/test_globalization.py

```diff
@@ -1,93 +1,156 @@
 # coding=utf-8
 import unittest
 import socket
-from utility import random_str, decode
+from utility import random_str
 from mssqltestutils import create_mssql_cli_client, shutdown
+from mssqlcli.mssqlcompleter import MssqlCompleter
+import mssqlcli.completion_refresher as completion_refresher
+from prompt_toolkit.document import Document
+
+
+"""
+All tests require a live connection to test server.
+Make modifications to mssqlutils.create_mssql_cli_client()
+to use a different server and database.
+Please Note: These tests cannot be run offline.
+"""
 
 
-# All tests require a live connection to test server.
-# Make modifications to mssqlutils.create_mssql_cli_client() to use a different server and database.
-# Please Note: These tests cannot be run offline.
 class GlobalizationResultSetTests(unittest.TestCase):
     
     def test_charset_double(self):
-        charset_list = self.get_charset_double() 
-        self.run_charset_validation(charset_list)
+        charset = GB18030()
+        characters_list = (charset.
+            get_next_characters(charset.get_double_characters()))
```

Comment:
I thought implicit line continuation are for parentheses, brackets and bracess. For this case I think the open paren seems more natural for readability.

Reply:
For example:

```
characters_list = charset.get_next_characters(
    charset.get_double_characters())
```

Reply:
fixed as requested.